app/views/schools/cohorts/show.html.erb

```diff
@@ -17,10 +17,20 @@
         </h2>
         <ul class="app-task-list__items">
 
+          <li class="app-task-list__item">
+            <span class="app-task-list__task-name">
+              <%= govuk_link_to "Choose the type of training",
+                edit_schools_cohort_programme_path(@cohort.start_year) %>
```

Comment:
Been told we need to use @cohort.display_name instead

Reply:
@amazimbe I think it makes sense to use `display_name` (view concern( when we display the cohort to a user. But we should be using data to build the urls (even though they are the same now!)